modules/utxo-lib/src/bitgo/index.ts

```diff
@@ -1,3 +1,5 @@
+export { PsbtInput, PsbtOutput } from 'bip174/src/lib/interfaces';
```

Comment:
```suggestion
export { PsbtInput, PsbtOutput } from 'bip174/;
```
Can we get to these without going into `src`?

Reply:
unfortunately not (hence the re-export here)